response.h

```diff
@@ -0,0 +1,97 @@
+// Represents an HTTP response.
+//
+// Usage:
+//   Response r;
+//   r.SetStatus(RESPONSE_200);
+//   r.SetBody(...);
+//   return r.ToString();
+//
+// Constructed by the RequestHandler, after which the server should call ToString
+// to serialize.
+
+
+#ifndef RESPONSE_H
+#define RESPONSE_H
+
+#include <boost/asio.hpp>
+#include <string>
+#include <vector>
+
+#define name first
+#define value second
+
+typedef std::pair<std::string, std::string> header;
+
+
+class Response {
```

Comment:
This file too since it was given to us the styling is off a bit with indentation and comments and whatnot.